repositories/rmw_implementation.BUILD.bazel

```diff
@@ -1,8 +1,28 @@
 """ Builds rmw_implementation.
 """
 
+load("@bazel_skylib//lib:selects.bzl", "selects")
+load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
 load("@com_github_mvukov_rules_ros2//ros2:cc_defs.bzl", "ros2_cpp_library")
 
+string_flag(
```

Comment:
Please move this and the two config settings to //ros2/BUILD.bazel. Where this is actually used is an implementation detail from user perspective, IMO.